src/buildercore/terraform.py

```diff
@@ -31,6 +39,13 @@ def render(context):
                         'ssl_cert_hostname': context['full_hostname'],
                         'ssl_check_cert': True,
                     },
+                    'gzip': {
+                        'name': context['stackname'],
```

Comment:
not sure what this `name` is used for, seems arbitrary from the documentation. Not a big deal to change if needed

Reply:
Tempted to just call it 'default'.